.github/workflows/ci.yml

```diff
@@ -25,15 +26,17 @@ jobs:
           - {VERSION: "3.10", TOXENV: "py310"}
           - {VERSION: "3.11", TOXENV: "py311"}
           - {VERSION: "3.12", TOXENV: "py312"}
-          - {VERSION: "3.12", TOXENV: "py312", NOTE: "system", SODIUM_INSTALL: "system"}
-          - {VERSION: "3.12", TOXENV: "py312", NOTE: "minimal", SODIUM_INSTALL_MINIMAL: "1"}
+          - {VERSION: "3.13", TOXENV: "py313"}
+          - {VERSION: "3.13", TOXENV: "py313", NOTE: "system", SODIUM_INSTALL: "system"}
+          - {VERSION: "3.13", TOXENV: "py313", NOTE: "minimal", SODIUM_INSTALL_MINIMAL: "1"}
     name: "Linux ${{ matrix.PYTHON.TOXENV }} ${{ matrix.PYTHON.NOTE }}"
     steps:
       - uses: actions/checkout@v4.2.0
       - name: Setup python
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.PYTHON.VERSION }}
+          allow-prereleases: true
```

Comment:
Our preference is not to use allow-pre-releases, but instead to specify 3.13-dev

Reply:
Updated!